builder/vsphere/iso/builder.go

```diff
@@ -60,6 +60,11 @@ func (b *Builder) Run(ctx context.Context, ui packer.Ui, hook packer.Hook) (pack
 		)
 	}
 
+	// default Firmware to "" since it is already configured by the CreateConfig.Firmware
+	if b.config.CreateConfig.Firmware != "" {
+		b.config.HardwareConfig.Firmware = ""
```

Comment:
Can we just remove the `b.config.HardwareConfig.Firmware` and all the related logic in `step_hardware`? 🤔

Reply:
that would work as long as we don't want it to change as part of the vsphere-clone

Reply:
or if we can move the logic into the StepCloneVM

Reply:
I think is ok to move the logic to StepCloneVM